### DISCUSSION TITLE
how to turn off gif and video to sticker

### DISCUSSION THREAD

**Discussion by dimanto019238** [kaiserdj/Stickerworld#43] [Q&A] [answered]
how to turn off gif and video to sticker,it makes the cpu be 100%

**Comment by kaiserdj** [comment]
I will add an option in the next version to disable the creation of stickers from gif or video or url, etc.

**Comment by kaiserdj** [comment, marked as answer]
Improvement included in the project: https://github.com/kaiserdj/Stickerworld/commit/e5a52ce949e6b052132aa8b272a475324903214e
